ARMTemplates/SnoozeVIS/azuredeploy.json

```diff
@@ -0,0 +1,792 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "LogicAppName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the logic app that creates start/stop schedule."
+      }
+    },
+    "SAPOperationType": {
+      "allowedValues": ["start", "stop"],
+      "type": "string",
+      "metadata": {
+        "allowedValues": ["start", "stop"],
+        "description": "Operation to be performed on SAP resource.",
+        "displayName": "SAP operation type",
+        "required": true
+      }
+    },
+    "IncludeDatabaseOperations": {
+      "type": "bool",
+      "metadata": {
+        "description": "Start/Stop the Database Instance",
+        "displayName": "Include Database instance",
+        "required": false
+      }
+    },
+    "IncludeVirtualMachineOperation": {
+      "type": "bool",
+      "metadata": {
+        "description": "Start/Stop the virtual machines",
+        "displayName": "Include virtual machines operation",
+        "required": true
+      }
+    },
+    "OperationExecution": {
+      "type": "string",
+      "metadata": {
+        "description": "Perform operations in sequence",
+        "displayName": "Perform operations in sequence",
+        "required": true
+      }
+    },
+    "SoftStopTimeout": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "required": false,
+        "displayName": "Soft stop timeout",
+        "description": "Soft stop timeout value, if this value is 0, hard stop will be triggered"
+      }
+    },
+    "VirtualInstanceForSAPSolutionsResourceID": {
+      "type": "array",
+      "metadata": {
+        "displayName": "Virtual Instance for SAP solutions Resource ID(s)",
+        "required": true
+      }
+    },
+    "Frequency": {

```

Comment:
These do not show up on UI

Reply:
If you are referring to At These Days/At these Hours, they are conditionally visible when certain "Frequency" is selected. Please let me know if you meant something else.